pylintrc

```diff
@@ -0,0 +1,646 @@
+[MAIN]
```

Comment:
I generated this with `pylint --generate-rcfile > pylintrc` and add private member access and non-top level imports to the ignored messages. The rest of the file is auto-generated defaults.